### ISSUE TITLE
requests version conflict with ibeam

### ISSUE THREAD

**Issue by blair-anson** [Voyz/ibind#16]
In a fresh python3.10 environment run
```
pip install ibeam
pip install ibind
```
The following error is displayed
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
ibeam 0.5.4 requires requests==2.30.*, but you have requests 2.31.0 which is incompatible.
```

I know many people run ibeam as a docker container so this is not an issue for them, but it would be nice to run both ibeam and ibind in the same python environment

**Comment by Voyz** [comment]
Thanks for reporting @blair-anson I'm currently travelling but I'll have this fixed ASAP 👍

**Comment by Voyz** [comment]
Hey @blair-anson I've released IBeam version `v0.5.5` which should have the same version of requests of 2.31.*. Let me know if it helps 👍

**Comment by Voyz** [comment]
I'm going to close this issue due to inactivity. Thanks for your contribution and please feel free to request a reopen if you'd like to continue the discussion 👍
